README.md

```diff
@@ -201,15 +207,15 @@ After the deployment is complete, you can try the deployed application by naviga
 
 Most of the Azure resources deployed in the prior steps will incur ongoing charges unless removed. Also a few of the resources deployed go into a soft delete status. It's best to purge those once you're done exploring, Key Vault is given as an example here. Azure OpenAI and Azure Machine Learning Workspaces are others that should be purged.
 
+> **_NOTE:_** To purge the AI services resources, `Microsoft.CognitiveServices/locations/resourceGroups/deletedAccounts/delete` permission must be given to the user on the subscription scope where the resources are deployed. A role assignment at the resource or resource group scopes will be insufficient to access the purge functionality. See the [learn documentation](https://learn.microsoft.com/en-us/azure/ai-services/recover-purge-resources?tabs=azure-cli#purge-a-deleted-resource) for additional details.
+
 ```bash
 az group delete --name $RESOURCE_GROUP -y
 
 # Purge the soft delete resources
-az keyvault purge -n kv-${BASE_NAME}
-az openai purge -n oai-${BASE_NAME}  <-- TODO
-az aml purge -n amlw <-- TODO
-az ai services purge -n aih-${BASE_NAME} <--- TODO
-Role assignment removals
+SUBSCRIPTION_ID=$(az account show --query id --output tsv)
+az keyvault purge -n kv-${BASE_NAME} -l $LOCATION --no-wait
+az resource delete --ids /subscriptions/${SUBSCRIPTION_ID}/providers/Microsoft.CognitiveServices/locations/${LOCATION}/resourceGroups/${RESOURCE_GROUP}/deletedAccounts/aih-${BASE_NAME}
```

Comment:
```suggestion
az resource delete --ids $(az cognitiveservices account show -n oai-${BASE_NAME} -g $RESOURCE_GROUP --query 'id' -o tsv)
```

And then remove `SUBSCRIPTION_ID`

Reply:
Maybe more importantly, why not just `az cognitiveservices account purge` ?

Reply:
So the recommended change may not work, because in theory the resource group delete would have already removed the resource so the az cognitiveservices account show shouldn't find it.  I'll try the purge option even though the official documentation suggests to use the resource delete option.  If it works I'll update to use the account purge as it's easier to read.